config/environment.js

```diff
@@ -94,7 +94,6 @@ module.exports = function(environment) {
         "packages_location",
         "stockit_contact",
         "goodcity_request",
-        "donor_condition",
```

Comment:
why removed donor_condition ?

Reply:
same question

Reply:
because the donor_condition is being unloaded from store. And it dosn't actually needs to be unloaded